game/platform.hh

```diff
@@ -25,12 +21,10 @@
 #endif
 
 struct Platform {
-enum platforms { windows, linux, macos, bsd, solaris, unix };
+enum class Host_OS { Performous_OS_Win32, Performous_OS_Linux, Performous_OS_macOS, Performous_OS_BSD, Performous_OS_Solaris, Performous_OS_Unix };
```

Comment:
You could remove the Platform struct (was just here to create a namespace) and do it
enum class Platform { OS_Win32, OS_Linux, OS_macOS, OS_BSD, OS_Solaris, OS_Unix };

Reply:
I think OS_Linux and OS_Win32 might also be defined by some compilers or in some circumstances, which is why I added Performous to the name also.

>

Reply:
Also, there are a few other things inside Platform. I don't think finding elsewhere to put them really warrants slightly shorter member names.

Reply:
Maybe Platform::OS::WIN32, PLATFORM:OS::LINUX etc?

Reply:
All caps is quite likely to collide with macros too. Win32, Linux, MacOS etc. might be safest if no prefix is used because traditional C coders don't like naming things like that.